crypto/ml_kem/ml_kem_ipd_ref_common/indcpa.c

```diff
@@ -53,9 +55,9 @@ static void unpack_pk(polyvec *pk,
 * Arguments:   - uint8_t *r: pointer to output serialized secret key
 *              - polyvec *sk: pointer to input vector of polynomials (secret key)
 **************************************************/
-static void pack_sk(uint8_t r[KYBER_INDCPA_SECRETKEYBYTES], polyvec *sk)
+static void pack_sk(ml_kem_params *params, uint8_t *r, polyvec *sk)
```

Comment:
`r` const?

Reply:
Could review rest of this file for more of these.

Reply:
I wanted to minimize changes in this PR and change only what is necessary so I skipped things like this. I'd really rather leave this as is and refactor/reimplement once we get more time allocated for it.